### PR TITLE
Dialog for displaying indirect member's groups implemented

### PR DESCRIPTION
**Pull request by xkostka2**
* Added dialog for displaying paths to member's direct groups
* Newest OpenAPI was generated

### CODE REVIEW

File: libs/perun/components/src/lib/members-list/members-list.component.ts [hunk 1]
```diff
@@ -170,6 +175,12 @@ export class MembersListComponent implements OnChanges, AfterViewInit {
     this.displayedColumns = this.displayedColumns.filter(x => !this.hideColumns.includes(x));
     this.dataSource = new MatTableDataSource<RichMember>(this.members);
     this.setDataSource();
+    this.disabledRouting = this.disableRouting;
```
Comment (balcirakpeter) [inline libs/perun/components/src/lib/members-list/members-list.component.ts:178]: I am just curious. Is this line needed?

**Review by Vojtech-Sassmann — CHANGES_REQUESTED** [review]
* The dialog should definitely be wider, the group names can take a lot of space, there is no point in making it this small.
* I don't understand the meaning of the House icon, or the men with arrows icon? I think you wanted to distinguish between a subgroup and group inclusion, but this is just confusing. There is no explanation, what they mean.

File: libs/perun/components/src/lib/members-list/members-list.component.ts (hunk 1 above)
Comment (xkostka2) [reply libs/perun/components/src/lib/members-list/members-list.component.ts:178]: disabledRouting is used to disable routing when clicking on a button located on a row

Comment (balcirakpeter) [reply libs/perun/components/src/lib/members-list/members-list.component.ts:178]: I know the purpose of the variable. What Ido not understand is why do you assign this variable to itself.

**Pull request closed without merging**